### PR TITLE
Fix: typos

### PR DESCRIPTION
**Pull request by omahs**
Fix: typos

### CODE REVIEW

File: contracts/dao-core/schema/dao-core.json [hunk 1]
```diff
@@ -49,7 +49,7 @@
         ]
       },
       "initial_items": {
-        "description": "The items to instantiate this DAO with. Items are arbitrary key-value pairs who's contents are controlled by governance.\n\nIt is an error to provide two items with the same key.",
+        "description": "The items to instantiate this DAO with. Items are arbitrary key-value pairs whose contents are controlled by governance.\n\nIt is an error to provide two items with the same key.",
```
Comment (0xekez) [inline contracts/dao-core/schema/dao-core.json:52]: for fixing this one, you'll need to modify this comment:

https://github.com/DA0-DA0/dao-contracts/blob/a3a45b9ae553529c526a4c01956fd673e7c684f7/contracts/dao-core/src/msg.rs#L46-L49

the code these json files is auto generated and CI will complain.

File: contracts/dao-core/schema/dao-core.json [hunk 2]
```diff
@@ -312,7 +312,7 @@
         "additionalProperties": false
       },
       {
-        "description": "Adds an item to the governance contract's item map. If the item already exists the existing value is overriden. If the item does not exist a new item is added.",
+        "description": "Adds an item to the governance contract's item map. If the item already exists the existing value is overridden. If the item does not exist a new item is added.",
```
Comment (0xekez) [inline contracts/dao-core/schema/dao-core.json:315]: https://github.com/DA0-DA0/dao-contracts/blob/a3a45b9ae553529c526a4c01956fd673e7c684f7/contracts/dao-core/src/msg.rs#L76-L78

File: contracts/dao-core/schema/dao-core.json [hunk 3]
```diff
@@ -1023,7 +1023,7 @@
                     ]
                   },
                   "channel_id": {
-                    "description": "exisiting channel to send the tokens over",
+                    "description": "existing channel to send the tokens over",
```
Comment (0xekez) [inline contracts/dao-core/schema/dao-core.json:1026]: this is, unfortunately, outside the domain of DAO DAO. you can make a PR to CosmWasm though if you would like to fix it!

https://github.com/CosmWasm/cosmwasm/blob/2eb97078418ca81a8dd1858871681ab97c4cd079/packages/std/src/ibc.rs#L30

**Review by 0xekez — COMMENTED** [review]
thanks!

we split our code into contracts, and their schemas. schemas are auto-generated from contracts and are the `.json` files that you modified. you'll need to modify the actual contract code to fix some of these and i've pointed to where you can find that in the code. :)

once you've modified the contract files, you can generate the schema files (without the typos) by running `just gen` from the repository root. you can find information about installing just [here](https://github.com/casey/just). tl;dr: `cargo install just`.

**Review by JakeHartnell — APPROVED** [review]

**Comment by omahs** [comment]
@0xekez Thanks for the elaborate answer! I added the changes in msg.rs.
I'm not a dev I try to contribute to various projects with docs, translations, reporting bugs and mostly use the web interface. If it's possible and you want to make the changes I can let you take over this PR

**Comment by 0xekez** [comment]
thanks @omahs! if you would ever like to contribute some code as well, i'd be happy to help get you started.

**Review by 0xekez — APPROVED** [review]

**Pull request merged**